src/main/java/cn/hashdata/bireme/pipeline/MaxwellPipeLine.java

```diff
@@ -28,8 +27,7 @@
 public class MaxwellPipeLine extends KafkaPipeLine {
   public MaxwellPipeLine(Context cxt, SourceConfig conf, int partitionID) {
```

Comment:
We do not bind partitionID to a pipeline anymore. It become useless and confused.